cas-server-support-oidc/src/main/java/org/apereo/cas/config/OidcConfiguration.java

```diff
@@ -106,6 +106,18 @@
     @Qualifier("defaultOAuthCodeFactory")
     private OAuthCodeFactory defaultOAuthCodeFactory;
 
+    @Autowired
+    @Qualifier("oidcAuthorizeEndpointController")
```

Comment:
This is an anti-pattern. All these should be removed and moved inside the bean itself. To fix the Controller annotation issue, try removing the BaseOAuthWrapperController as a return type and return the actual bean type.